smokefree/everytrycounts/index.html

```diff
@@ -12,26 +12,6 @@
 <title>Every Try Counts</title>
 <!-- Disable tap highlight on IE -->
 <meta name="msapplication-tap-highlight" content="no">
-	<!-- Google Tag Manager -->
-	<script>
```

Comment:
Are we intentionally removing this GTM code? Has this been approved by Brian Keefe?